src/network/message.rs

```diff
@@ -70,7 +70,7 @@ impl Encodable for CommandString {
         let mut rawbytes = [0u8; 12];
         let strbytes = self.0.as_bytes();
         if strbytes.len() > 12 {
-            return Err(encode::Error::UnrecognizedNetworkCommand(self.0.clone().into_owned()));
+            return Err(encode::Error::ParseFailed("invalid command string"));
```

Comment:
This error case is probably going away with https://github.com/rust-bitcoin/rust-bitcoin/pull/494 anyway.